retriever/try_install_modified.py

```diff
@@ -0,0 +1,139 @@
+"""Attempt to update datasets in all database management systems.
+This module, when run, attempts to install datasets from modified Retriever
+scripts in the /scripts folder (except for those listed in IGNORE), for each
+engine in ENGINE_LIST() from __init__.py. In other words, it trys to install
+using all possible combinations of database platform and script and checks
+to see if there are any errors. It does not check the values in the database.
+"""
+from __future__ import print_function
+from __future__ import absolute_import
+import os
+import sys
+from imp import reload
+from retriever.lib.tools import choose_engine
+from retriever import MODULE_LIST, ENGINE_LIST, SCRIPT_LIST
+from retriever.version import get_module_version, update_version_file
+
+
+def get_modified_scripts():
+    """Get modified script list compared to version.txt."""
+    modified_scripts = []
+    script = get_module_version()
+    with open("version.txt", "r") as version_file:
+        version_file.readline()
+        for counter, line in enumerate(version_file.readlines()):
```

Comment:
I might have gotten the relative directories wrong. Can you try running this sequence of statements in the terminal: 

```
git fetch https://github.com/weecology/retriever.git master 2>/dev/null
git diff --name-only FETCH_HEAD scripts
```
to see what gets outputted? The first one should output nothing, and the second one should output changed scripts, so you can test it by modifying some scripts and then running those two commands. If that works, change the lines to that in the file and try running it again. That should work.